### PR TITLE
Telcodocs-230 Updating Performing end-to-end tests for platform verify

### PR DESCRIPTION
**Pull request by kquinn1204**
Initial update. Link to JIRA https://issues.redhat.com/browse/TELCODOCS-230
The issue applies to 4.8 and does not need merged into 4.7.

### CODE REVIEW

**Comment by kquinn1204** [comment]
published draft is https://deploy-preview-31203--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-for-platform-verification_cnf-master

**Comment by kquinn1204** [comment]
@lack Hi in line with the changes you made in https://github.com/openshift-kni/cnf-features-deploy/pull/467 I updated the docs for 4.8 in this PR. I would be grateful if you could review to ensure I did not miss anything.

I took my lead from https://github.com/openshift-kni/cnf-features-deploy/pull/467/files

**Review by lack — APPROVED** [review]

**Comment by kquinn1204** [comment]
Hi @ahardin-rh @lack has approved what I have added to the Performing end-to-end tests for platform verification. I don't believe there is any need to run past QE as there are no significant procedural changes. I would be grateful if you could merge.

File: modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc [hunk 1]
```diff
@@ -626,6 +632,11 @@ Performance tests apply a performance profile to the cluster. The effect of this
 
 DPDK relies on both performance and SR-IOV features, so the test suite configures both a performance profile and SR-IOV networks, so the impacts are the same as those described in SR-IOV testing and performance testing.
 
+[id="cnf-performing-end-to-end-tests-container-mount-namespace_{context}"]
+=== Container-mount-namespace
+
+The validation test for container-mount-namespace mode only checks that the appropriate MachineConfig objects are present and active, and has no additional impact on the node.
```
Comment (ahardin-rh) [inline modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc:638]: ` container-mount-namespace` mode

    `MachineConfig` objects

**Comment by ahardin-rh** [comment]
@kquinn1204 What versions does this apply to? Just 4.8? Can you please put a link to the JIRA card in the first comment so that we have a paper trail? 

Just a few minor style comments from me. Thank you!

**Comment by kquinn1204** [comment]
@ahardin-rh changes made thanks for the review. The issue applies to 4.8 and does not need to be merged into 4.7.

**Comment by kquinn1204** [comment]
@ahardin-rh there is no release notes needed here so this can be merged to 4.8. It does not need merged to 4.7

**Pull request merged**